doc/source/user_guide/visualization.rst

```diff
@@ -1369,6 +1377,7 @@ Another option is passing an ``ax`` argument to :meth:`Series.plot` to plot on a
 .. ipython:: python
 
    fig, axes = plt.subplots(nrows=2, ncols=2)
+   plt.subplots_adjust(wspace=0.2, hspace=0.4)
```

Comment:
Nice! Sorry for the nitpick, but could hspace be increased slightly more so there's visual separation between the labels of the top subplots and the titles of the bottom ones?

Reply:
Of course!